packages/actor-rdf-resolve-quad-pattern-hypermedia/lib/MediatedQuadSource.ts

```diff
@@ -15,19 +15,21 @@ export class MediatedQuadSource implements IQuadSource {
   public readonly context: IActionContext;
   public readonly firstUrl: string;
   public readonly forceSourceType?: string;
+  public readonly ignoreSparqlEndpointFromServiceDescription?: boolean;
   public readonly mediators: IMediatorArgs;
 
   public sourcesState: ISourcesState;
 
   private readonly cacheSize: number;
   private readonly maxIterators: number;
 
-  public constructor(cacheSize: number, context: IActionContext, firstUrl: string,
-    forceSourceType: string | undefined, maxIterators: number, mediators: IMediatorArgs) {
+  public constructor(cacheSize: number, context: IActionContext, firstUrl: string, forceSourceType: string | undefined,
+    ignoreSparqlEndpointFromServiceDescription: boolean | undefined, maxIterators: number, mediators: IMediatorArgs) {
```

Comment:
The changes here won't be needed anymore when using the context to store this information.